changelog/63.breaking.2.rst

```diff
@@ -0,0 +1,6 @@
+Deprecated the following pfss utils:
+
+- pfss.utils.fix_hmi_meta
+- pfss.utils.load_adapt
+
+These functions are no longer needed and will be removed.
```

Comment:
Are there alternatives we can direct people to?

Reply:
There is an example on how to load the adapt maps. FOr the fix meta, we have the how to. 

I can link to both.